sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -30,22 +32,19 @@ public async Task RunSampleAsync()
             ModuleIdentity moduleIdentity = await CreateModuleIdentityAsync(deviceIdentity, moduleId);
```

Comment:
nit - afaik, these are [IoT hub modules](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-module-twins), and not [edge modules](https://docs.microsoft.com/en-us/azure/iot-edge/iot-edge-modules). We should confirm that, and if true, call that out in our comments.